RELEASE_NOTES.md

```diff
@@ -0,0 +1,39 @@
+[comment]: # (#################################################################)
+[comment]: # (Copyright 2017-22, Lawrence Livermore National Security, LLC)
+[comment]: # (Produced at the Lawrence Livermore National Laboratory.)
+[comment]: # (All rights reserved.)
+[comment]: # 
+[comment]: # (#################################################################)
+
+Version vxx.yy.zz -- Release date 20yy-mm-dd
+============================================
+
+This release contains ...
+
+Notable changes include:
+
+  * Dependency updates:
+
+  * New changes/improvements:
+
+  * Build changes/improvements:
+
+  * Bug fixes/improvements:
+
+
+Version 1.1.0 -- Release date 2022-06-15
+============================================
+
+This release contains new features, bug fixes, and build improvements.
+
+Notable changes include:
+
+  * Dependency updates:
+      * RAJA    87a5cac67 (v2022.03.0 with additional fixes)
+      * BLT     v0.5.1
+      * Kripke  v1.2.6
+            * Umpire  v2022.03.1
+            * CHAI    v2022.03.0
+
+  * New changes/improvements:
+      * Added HIP backend to CoMD, and Lulesh.
```

Comment:
@davidbeckingsale @rhornung67 Hopefully this is the correct way for the release notes to show up? I'm just using the format from the RAJA repo.

Reply:
@rchen20 you will have to copy the notes in the file to the release summary on GitHub.